### PR TITLE
[Ubuntu] Add Microsoft Edge

### PR DESCRIPTION
**Pull request by mikhailkoliada**
# Description

Add Microsoft Edge to all Ubuntu's versions

## Check list
- [X] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/tests/Browsers.Tests.ps1 [hunk 1]
```diff
@@ -18,8 +18,16 @@ Describe "Chrome" {
...
+    }
+
+}
+
 Describe "Chromium" {
     It "Chromium" {
         "chromium-browser --version" | Should -ReturnZeroExitCode
     }
-} 
+
```
Comment (al-cheb) [inline images/linux/scripts/tests/Browsers.Tests.ps1:32]: nitpick: remove empty line

**Review by al-cheb — APPROVED** [review]

File: images/linux/scripts/installers/microsoft-edge.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+REPO_URL="https://packages.microsoft.com/repos/edge"
+
+wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /usr/share/keyrings/microsoft-edge.gpg
+# Specify an arch as Microsoft repository supports armhf and arm64 as well
+echo "deb [arch=amd64 signed-by=/usr/share/keyrings/microsoft-edge.gpg] $REPO_URL stable main" > /etc/apt/sources.list.d/microsoft-edge.list
+
+apt-get update
+apt-get install --no-install-recommends microsoft-edge-stable
+
+rm /usr/share/keyrings/microsoft-edge.gpg
```
Comment (al-cheb) [inline images/linux/scripts/installers/microsoft-edge.sh:16]: use variables for the /usr/share/keyrings/microsoft-edge.gpg and /etc/apt/sources.list.d/microsoft-edge.list

File: images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1 [hunk 3]
```diff
@@ -23,6 +23,12 @@ function Get-ChromiumVersion {
     return $chromiumVersion
 }
 
+function Get-EdgeVersion {
+    $edgeVersion = microsoft-edge --version | Take-OutputPart -Part 2
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1:27]: Remove | Take-OutputPart -Part 2

File: images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1 [hunk 4]
```diff
@@ -23,6 +23,12 @@ function Get-ChromiumVersion {
...
+    $aptSourceRepo = Get-AptSourceRepository -PackageName "microsoft-edge"
+    return "Microsoft Edge $edgeVersion (apt source repository: $aptSourceRepo)"
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1:29]: Remove Microsoft Edge

**Review by ddobranic — DISMISSED** [review]

**Review by al-cheb — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1 [hunk 5]
```diff
@@ -23,6 +23,12 @@ function Get-ChromiumVersion {
     return $chromiumVersion
 }
 
+function Get-EdgeVersion {
+    $edgeVersion = microsoft-edge --version
+    $aptSourceRepo = Get-AptSourceRepository -PackageName "microsoft-edge"
+    return "$edgeVersion (apt source repository: $aptSourceRepo)"
```
Comment (miketimofeev) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1:29]: looks like we don't need whitespace here
image

Comment (mikhailkoliada) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1:29]: trimmed it, lets go again

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**